widgets/GCSMultiFiles-batchsink.json

```diff
@@ -0,0 +1,138 @@
+
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "Multi Files Google Cloud Storage Sink",
```

Comment:
remove 'Sink', as the plugin will already show up in the sinks section in the UI.